share/containers/fedora.dockerfile

```diff
@@ -15,16 +15,19 @@ RUN ./autogen.sh \
 	--enable-shadowgrp \
 	--enable-man \
 	--with-audit \
-        --with-sha-crypt \
+	--with-libpam \
+	--with-sha-crypt \
 	--with-bcrypt \
 	--with-yescrypt \
 	--with-selinux \
-        --without-libpam \
-	--enable-shared \
 	--without-libbsd \
-        --with-group-name-max-length=32 \
+	--without-libcrack \
+	--without-sssd \
+	--enable-shared \
+	--with-group-name-max-length=32 \
 	--enable-lastlog \
-	--enable-logind=no
+	--enable-logind=no \
+	--disable-account-tools-setuid
```

Comment:
Oh, and there are also `--enable-*` ones up there.  Can we have them all together?  Or do you have reasons to prefer them in a different order?

Reply:
I would use whatever order sort(1) produces.

Reply:
Updated